File: ArrayQuestion/src/LinearSearch.java
public class LinearSearch {

    public static void main(String[] args){

        int[] arr = {6,7,9,4,1,0};
        int target = 0;

       System.out.println("Target fount at index " + LinearSearch(arr,target));
    }

    static int LinearSearch(int[] arr, int target){

        for(int i = 0; i < arr.length; i++){
            if(arr[i] == target){
                return i;
            }
        }

        return 0;
    }

}

File: ArrayQuestion/LinearSearch.cpp
#include <iostream>

using namespace std;

int LinearSearch(vector<int> arr, int target){

    for(int i = 0; i < arr.size(); i++){
            if(arr[i] == target){
                return i;
            }
        }

        return -1;
}

int main(){

vector<int> arr = {5,5,7,2,7};
int target = 7;

cout<<"Target found at index "<<LinearSearch(arr,target);
return 0;
}